src/main/java/com/github/legoatoom/connectiblechains/client/render/entity/ChainRenderer.java

```diff
@@ -18,96 +18,74 @@
 package com.github.legoatoom.connectiblechains.client.render.entity;
 
 import com.github.legoatoom.connectiblechains.ConnectibleChains;
+import com.github.legoatoom.connectiblechains.chain.UVRect;
 import com.github.legoatoom.connectiblechains.util.Helper;
 import it.unimi.dsi.fastutil.objects.Object2ObjectOpenHashMap;
+import net.fabricmc.loader.api.FabricLoader;
 import net.minecraft.client.render.VertexConsumer;
 import net.minecraft.client.util.math.MatrixStack;
-import net.minecraft.util.math.*;
+import net.minecraft.util.math.Quaternion;
+import net.minecraft.util.math.Vec3d;
+import net.minecraft.util.math.Vec3f;
 
 import static com.github.legoatoom.connectiblechains.util.Helper.drip2;
 import static com.github.legoatoom.connectiblechains.util.Helper.drip2prime;
 
 public class ChainRenderer {
-    private final Object2ObjectOpenHashMap<BakeKey, ChainModel> models = new Object2ObjectOpenHashMap<>(256);
```

Comment:
This class is the big part of the mod, we need clear documentation here. Rendering is already hard enough to begin with. Assume the reader does not know about baking. (Not because I don't know what baking is... noo that would be bad if I didn't know _that_)

Reply:
Yes I did kind of miss that class when writing documentation.